apps/mempool/docker-compose.yml

```diff
@@ -44,7 +44,7 @@ services:
       MYSQL_DATABASE: "mempool"
       MYSQL_USER: "mempool"
       MYSQL_PASS: "mempool"
-      CACHE_DIR: "/backend/cache/"
```

Comment:
Why is this change needed?

Reply:
They've update mempool so now their code add this var to "/cache.json" so I've removed the useless slash on the docker-compose

Reply:
Ahh, good spot!